content/attacks/posts/2019-06-02-Kraken.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2019-06-02
+target-entities: Kraken
+entity-types:
+  - Custodian
+  - Exchange
+attack-types: Market Manipulation
+title: "Kraken Bitcoin Flash Crash to $100 CAD: Potential Hacker Activity"
+loss: 10470000
+---
+
+## Summary
+
+On June 2, 2019, Kraken, a respected cryptocurrency exchange, experienced a flash crash on its Bitcoin-to-Canadian Dollar (BTC/CAD) trading pair. In a matter of seconds, the price of BTC dropped from $11,200 CAD to just over $101 CAD, a decline of over 99%. The incident resulted in losses of approximately $10,470,000 USD, equivalent to 1,200 BTC. Analysts suggest that this event was executed by a hacker attempting to siphon funds from Kraken. The attacker is alleged to have executed the flash crash using a specific method. Firstly, they managed to gain access to an account that held 1,200 BTC, but they were unable to withdraw the funds due to two-factor authentication. To maneuver around this, they set up their own buy orders, capping them at $100 CAD, on the BTC/CAD trading pair, which was not frequently traded. Then they sold off the entire 1,200 BTC into this illiquid market. As a result of this move, they effectively transferred the funds to themselves. Through this strategy, the attacker managed to become the legitimate owner of the 1,200 BTC, securing them at a price considerably below the prevailing market rates.
+
+## Attackers
+
+The identity of the attacker remains unknown.
+
+## Losses
+
+Kraken experienced losses of around $10,470,000 USD worth of 1,200 BTC.
```

Comment:
"...losses of 1,200 BTC, worth around $10,470,000 USD at the time of the incident"